apps/coscrad-frontend/src/utils/generic-components/presenters/multilingual-text-presenter.tsx

```diff
@@ -22,37 +20,29 @@ export const MultilingualTextPresenter = ({
     const textItemWithDefaultLanguage =
         items?.find((item) => item.languageCode === defaultLanguageCode) || null;
 
+    const primaryMultilingualTextItem = isNullOrUndefined(textItemWithDefaultLanguage)
+        ? items?.find((item) => item.role === MultilingualTextItemRole.original)
```

Comment:
You need to use existing or build new unit-tested utilities for doing this work.

Reply:
getMultilingualTextItemByRole(multilingualText: IMultilingualText): Maybe<IMultilingualTextItem> // + a unit test

Reply:
getMultilingualTextItemByLanguage(multilingualText: IMultilingualText, languageCode: LangaugeCode): Maybe<IMultilingualTextItem>// + a unit test